apps/dashboard/app/helpers/system_status_helper.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+# helper for the system status page
+module SystemStatusHelper
+  # Used to not repeatedly request the job adapter
+  def update_cluster(cluster)
+    @job_adapter = cluster.job_adapter
+    @cluster_info = @job_adapter.cluster_info
+    @cluster = cluster
```

Comment:
We don't want member variables in helpers. They should be treated as functional methods - i.e., they don't hold any state, are idempotent and so on.